File: backend/src/main/java/com/example/conferencemanagementsystem/model/Section.java
package com.example.conferencemanagementsystem.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.OneToOne;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Section {
    @Id
    @GeneratedValue
    private int id;
    @OneToOne
    private Conference conference;
    @OneToOne
    private ProgramCommitteeMember mainChair;
    private String room;
    private int noSeats;
    private int listenerFee;
}

File: webapp/src/app/model/section.ts
import {Conference} from "./conference";
import {ProgramCommitteeMember} from "./program-committee-member";

export class Section {
  id: number;
  conference: Conference;
  mainChair: ProgramCommitteeMember;
  room: string;
  noSeats: number;
  listenerFee: number;
}
